sample_files/.travis.yml

```diff
@@ -11,9 +11,21 @@ addons:
 
 env:
   global:
-  - VERSION="8.0" TESTS="0" LINT_CHECK="0"
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  # This line contains the encrypted transifex password
+  # To encrypt transifex password, use
+  # $ travis encrypt TRANSIFEX_PASSWORD=your-password -r owner/project
```

Comment:
No, it was my own comment, that I put transifex instead of Travis. OK, I make the PR. I'm configuring Transifex for l10n-spain. I'm doing with my own account. Can you give transbot credentials or I continue with current?